geth/wrapper.py

```diff
@@ -249,6 +256,9 @@ def construct_popen_command(
     if miner_threads is not None:
         if not mine:
             raise ValueError("`mine` must be truthy when specifying `miner_threads`")
+        logging.warning(
```

Comment:
Maybe this should just be a comment in the code but open to suggestions.

Reply:
logging seems fine to me